nature/benchmarks/linear_mapper_benchmark.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Linear Mapper Benchmarks."""
+from timeit import timeit
+import numpy as np
+
+import retworkx
+from qiskit_nature.mappers.second_quantization import LinearMapper
+from qiskit_nature.problems.second_quantization.lattice.models import IsingModel
+from qiskit_nature.problems.second_quantization.lattice import Lattice
+
+# pylint: disable=redefined-outer-name, invalid-name, attribute-defined-outside-init
+
+
+class LinearMapperBenchmarks:
+    """Linear Mapper Benchmarks."""
+
+    version = 1
+    seed = 100
+    params = list(range(3))
+    param_names = ["op_number"]
+
+    def setup_cache(self):
+        """setup cache"""
+
+        second_q_ops_list = []
+
+        for _ in range(3):
+            num_nodes = np.random.randint(40)
```

Comment:
I just noticed this. We should avoid using randomness in benchmarks because we want to compare performance on the very same instances every time (it seems that also `seed` is not used anywhere). I would suggest constructing some reasonable graphs manually. Each graph might have a different number of nodes and you can name each example by the number of nodes (this refers to the comment about making the benchmarks omre descriptive). Another option would be to use some known classes of graphs and create them through the retworkx methods as long as they are deterministic.

Reply:
Might it not actually make more sense to benchmark increasing sizes in terms of `num_modes`?